__tests__/Unit/utils/getActiveTab.test.ts

```diff
@@ -14,4 +14,8 @@ describe('Unit | Util | Get Active Tab', () => {
         expect(getActiveTab('in-progress')).toEqual(Tab.IN_PROGRESS);
         expect(getActiveTab('someRandomSection')).toEqual(Tab.ALL);
     });
+
+    test('returns proper value for the overdue tab when active', () => {
+        expect(getActiveTab('overdue')).toEqual(Tab.OVERDUE);
+    });
```

Comment:
are you sure this test is passing without rendering the component?

Reply:
yes, i have written test separately to check the rendering of overdue tab separately.

Reply:
so why is it written here, please remove it then

Reply:
and also dont resolve comment until reviewers confirmation over the comment